transcom/mymove
# [ MB-16480 ] Update error page contact information

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16480)

## Summary

This patch updates the **"Oops..."** error page with updated contact
information. To best test this, you will need to reproduce an error on the
client-side in order to see this page.

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Login as a TOO user and go to the Move Queue.
1. Change the URL to match the following example.
   - `https://office.stg.move.mil/moves/showerrorpage/details`
   - Note the URL path in the example, `/moves/showerrorpage/details`.
1. See the new **"Oops..."** page with updated contact information.
